### PR TITLE
Make debowerify work with more than 2 entries

### PR DESCRIPTION
**Pull request by microadam**
Apologies! Oversight from last PR!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -141,9 +141,9 @@ module.exports = function (file, options) {
       for (var i = node.range[0] + 1; i < node.range[1]; i++) {
           chunks[i] = '';
       }
-      paths.forEach(function (p) {
+      paths.forEach(function (p, i) {
         var st = '\nrequire(' + p + ')'
-        chunks[node.range[1] + 1] = st
+        chunks[node.range[1] + (i + 1)] = st
```
Comment (bobthecow) [inline index.js]: do we have to worry about this running out of chunks to overwrite?

**Comment by eugeneware** [comment]
I'm seeing a failed test. I think `c.js` needs to be added?

``` js
 Σ debowerify master$ npm test

> debowerify@1.3.1 test /Users/eugeneware/Dropbox/work/debowerify
> mocha


  ․

  0 passing (687ms)
  1 failing

  1) debowerify should be able to debowerify a module with multiple main entries:
     Error: module "./../bower_components/test-package-c/c.js" not found from "/Users/eugeneware/Dropbox/work/debowerify/public/multiple_main_entries.js"
      at notFound (/Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/index.js:744:15)
      at /Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/index.js:695:23
      at /Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/node_modules/browser-resolve/index.js:185:24
      at /Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/node_modules/resolve/lib/async.js:36:22
      at load (/Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/node_modules/resolve/lib/async.js:54:43)
      at /Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/node_modules/resolve/lib/async.js:60:22
      at /Users/eugeneware/Dropbox/work/debowerify/node_modules/browserify/node_modules/resolve/lib/async.js:16:47
      at Object.oncomplete (fs.js:107:15)
```

**Comment by microadam** [comment]
All commited! Missed those, doing things in a rush always ends up causing issues! Should all be good now

File: index.js (hunk 1 above)
Comment (microadam) [inline index.js]: I dont believe so, if the chunks array gets exhausted, it should just append to the end of it?

Comment (bobthecow) [inline index.js]: i think if it hits the end of the empty chunks it'll start overwriting other code.

**Comment by bobthecow** [comment]
@maxnordlund There was an unresolved concern above. I'm pretty sure this will start overwriting code once it runs out of emptied out chunks.

**Comment by maxnordlund** [comment]
Darn it, I totally missed that one. I'm so sorry @bobthecow. Should we revert, or just not bump the version number yet?

**Comment by bobthecow** [comment]
Depends on how fast we can get a fix in :)

**Comment by mkoryak** [comment]
oh, this isnt fixed. i see.

**Pull request merged**